support/regression-test.tex

```diff
@@ -293,6 +293,25 @@
   \fi
 \fi
 \reset@catcodes
-%% 
+%%
+
+%%% this needs to be moved to the source of regression-test.tex!
+
+\ifx\AddToHook\undefined
```

Comment:
I assume for the 'real thing' this should use the LaTeX release date?

Reply:
Maybe I misunderstand you, but I don't think so. Point is you can use regression-test.tex wiht formats other than LaTeX, don't you? So we need a test that does a) is it LaTeX and b) is it new enough? So I think the above is a simple way to do both.